### ISSUE TITLE
Editable Reblogs: Post disappears when editing in HTML Editor

### ISSUE THREAD

**Issue by ofinnocxnce** [new-xkit/XKit#585]
I have everything updated, all the extensions and stuff... So the problem is, when I reblog a post, add something to it myself and edit it in the HTML Editor and then reblog it while staying in the HTML editor, what the other person said before completely disappears and the only thing that stays is what I added. This doesn't happen in the Rich Editor, only the HTML one. I also already saw this happening to a lot of other people on my dash. 
If you need an example of how it looks like, let me know and I'll show that to you.

**Comment by AuraSweet** [comment]
It happens similarly for me, except it's just on posts I reblog in general. I also have proof if you want to see it.

**Comment by Wolvan** [comment]
Did anyone of you two edit the Username by any chance? Make it bold, remove the colon, anything at all?

**Comment by AuraSweet** [comment]
I tried to, but the post still disappeared anyway.

**Comment by Wolvan** [comment]
Did you edit the blog title in any way the first time around?

**Comment by AuraSweet** [comment]
The post title? No, I don't think so.

**Comment by Wolvan** [comment]
Sorry, I meant the username somewhere in the reblog chain

**Comment by AuraSweet** [comment]
I'm pretty sure I didn't.

**Comment by Wolvan** [comment]
Can you link us to the post? And tell us what you did step by step to reproduce this problem?

**Comment by AuraSweet** [comment]
I couldn't get support for a while, so I implemented other things to fix it, so I'm not sure how to reproduce it. :$

**Comment by AuraSweet** [comment]
I'll try, though.

**Comment by Wolvan** [comment]
what DID you implement? That might be good to know as well

**Comment by AuraSweet** [comment]
![problem](https://cloud.githubusercontent.com/assets/14351360/10273002/67057bc6-6ae6-11e5-948b-66fa59979a34.jpg)

**Comment by AuraSweet** [comment]
I implemented Tampermonkey just to make things back to normal until I could get Xkit.

**Comment by Wolvan** [comment]
Can you link us to that post? And what tampermonkey script exactly?

**Comment by AuraSweet** [comment]
http://confidxnteveryday.tumblr.com/post/130529127207/scxrletlass

**Comment by AuraSweet** [comment]
When I reblogged that, I had Tampermonkey disabled, but it was the tumblr Reblog Fix.

**Comment by AuraSweet** [comment]
When I reblogged the post I just linked you, which disappears on the dash, I had Editable Reblogs enabled.

**Comment by AuraSweet** [comment]
There's two of them, not just the one, if that helps any :D

**Comment by AuraSweet** [comment]
https://greasyfork.org/en/scripts/11318-tumblr-reblog-fix
https://greasyfork.org/en/scripts/12164-tumblr-reblog-fix-two/code
And they only work if you have Chrome, for Firefox I think it's different.

**Comment by AuraSweet** [comment]
But I had both of those disabled and ER enabled when the post above happened.

**Comment by AuraSweet** [comment]
So I thought it might be Xkit and came to you guys.

**Comment by ofinnocxnce** [comment]
Yeah, I personally did edit the URL of the person the post was originally from like making it into a header or bold or whatever, that's what I did and for a friend of mine it even happens when she doesn't edit the posts in the html editor, but just in the rich editor and edits the url there
